int16 drone_id
float32 lat_home
float32 lon_home
float32 lat_predict
float32 lon_predict
bool moving_flag